customizations/README.md

```diff
@@ -454,35 +487,91 @@ appConfigValues:
   workspaceImportsBucket: "carto-client-bucket"
   workspaceThumbnailsBucket: "carto-thumbnails-bucket"
   workspaceThumbnailsPublic: false
+  awsS3Region: "us-east-1"
+  
+```
+For the secrets, use **one** of the following options:
 
+**Option 1: Automatically create the secret**
+
+Add the following lines to your `customizations.yaml`:
+```yaml
 appSecrets:
   awsAccessKeyId:
     value: "<REDACTED>"
   awsAccessKeySecret:
     value: "<REDACTED>"
 ```
-
 > `appSecrets.awsAccessKeyId.value` and `appSecrets.awsAccessKeySecret.value` should be in plain text
 
-### Azure Storage
+**Option 2: Using existing secret**
+Create a secret with your awsAccessKeyId and awsSecretAccessKey
+```bash
+kubectl create secret generic \                                                                      
+  [-n my-namespace] \
+  mycarto-custom-s3-secret \
+  --from-literal=awsAccessKeyId=<REDACTED> \
+  --from-literal=awsSecretAccessKey=<REDACTED>
+```
+> Use the same namespace where you are installing the helm chart
 
 Add the following lines to your `customizations.yaml`:
 
+```yaml
+appSecrets:
+  awsAccessKeyId:
+    existingSecret:
+      name: mycarto-custom-s3-secret
+      key: awsAccessKeyId
+  awsAccessKeySecret:
+    existingSecret:
+      name: mycarto-custom-s3-secret
+      key: awsSecretAccessKey
+```
+### Azure Storage
+
+Add the following lines to your `customizations.yaml` change the name of the buckets with your own settings:
+
 ```yaml
 appConfigValues:
   storageProvider: "azure-blob"
   importBucket: "carto-import-bucket"
   workspaceImportsBucket: "carto-client-bucket"
   workspaceThumbnailsBucket: "carto-thumbnails-bucket"
   workspaceThumbnailsPublic: false
+```
+For the secrets, use **one** of the following options:
+
+**Option 1: Automatically create the secret**
 
+```yaml
 appSecrets:
   azureStorageAccessKey:
     value: "<REDACTED>"
 ```
 
 > `appSecrets.azureStorageAccessKey.value` should be in plain text
 
+**Option 2: Using existing secret**
+Create a secret with your azureStorageAccessKey
```

Comment:
`azureStorageAccessKey` ?

Reply:
yes, https://docs.microsoft.com/en-us/azure/storage/common/storage-account-keys-manage?tabs=azure-portal regarding azure. I cna change to "Azure Storage Account Key" if it sounds better